examples/interaction/js_callbacks/customjs_for_hover.py

```diff
@@ -1,3 +1,15 @@
+''' An interactive plot showcasing Bokeh's ability to add interactions
+using Custom Javascript.
+This example demonstrates adding links between points on a graph.
```

Comment:
```suggestion
using Custom Javascript.

This example demonstrates adding links between points on a graph.
```
similar blank line between paragraphs